lib/build.js

```diff
@@ -192,22 +195,19 @@ class BuildModel extends BaseModel {
             return nodeify.success(null);
         }
 
+        // We are aborting the build here, setting the status so SCM updated properly
+        this.status = 'ABORTED';
+
         // Make sure that a pipeline and job is associated with the build
         return this.pipeline
             // stop the build
             .then(pipeline =>
+                // Launcher should notify the builds webhook to update
+                // status and endTime when actually complete
                 nodeify.withContext(this[executor], 'stop', [{ buildId: this.id }])
-                // get the admin for notification
-                .then(() => pipeline.admin)
                 // update github
-                .then(admin =>
-                    updateSCM({
-                        user: admin,
-                        scmUrl: pipeline.scmUrl,
-                        sha: this.sha,
-                        state: 'failure'
-                    })
-                )
+                .then(() => this.updateCommitStatus(pipeline))
```

Comment:
When the launcher notifies the builds webhook, it should also be updating the commitStatus.

Is there a need to do it here as well?